flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -252,3 +252,31 @@ def mach(self) -> pd.PositiveFloat:
 # pylint: disable=fixme
 # TODO: AutomotiveCondition
 OperatingConditionTypes = Union[GenericReferenceCondition, AerospaceCondition]
+
+
+def operating_condition_from_mach_reynolds(
+    mach: pd.NonNegativeFloat = None,
+    reynolds: pd.PositiveFloat = None,
+    temperature: pd.PositiveFloat = None,
+    alpha: Optional[AngleType] = 0 * u.deg,
+    beta: Optional[AngleType] = 0 * u.deg,
+    reference_mach: Optional[pd.PositiveFloat] = None
+) -> AerospaceCondition:
+    thermal_state = ThermalState(
+        temperature=temperature,
+        material=Air(
+            dynamic_viscosity=Sutherland(
+                reference_temperature=temperature,
+                reference_viscosity=(mach/reynolds),
+                effective_temperature=110.4
+            )
+        )
+    )
```

Comment:
what units do we use here? I think this is not expected, it will infer units from the used unit system, thus resulting in different mach/reynolds values. I would expect this here:

```suggestion
    thermal_state = ThermalState(
        temperature=temperature,
        material=Air(
            dynamic_viscosity=Sutherland(
                reference_temperature=temperature,
                reference_viscosity=(mach/reynolds) * u.flow360_viscosity_unit,
                effective_temperature=110.4 * u.K
            )
        )
    )
```